bootbox.js

```diff
@@ -216,6 +216,24 @@
    * merge a set of default dialog options with user supplied arguments
    */
   function mergeArguments(defaults, args, properties) {
+    var mapArgumentsResult = mapArguments(
+      args,
+      properties
+    );
+    if(defaults && defaults.buttons && mapArgumentsResult && mapArgumentsResult.buttons){
```

Comment:
Thanks for reporting the issue. Using the `grunt` command it's all fine (apart from the `getKeyLength` function). What are you referring to when you talk about code style?